check50.py

```diff
@@ -200,6 +201,11 @@ def excepthook(cls, exc, tb):
 sys.excepthook = excepthook
 
 
+def handler(number, frame):
+    print()
+    sys.exit(1)
+
+
```

Comment:
This seems to omit much of the handling we baked into `submit50`? Could we instead invoke the existing `handler` in `submit50.py` by exporting it (and parameterizing its `cprint` call to be "Submission cancelled." or "Checking cancelled."?

```
def handler(number, frame):
    """Handle SIGINT."""
    os.system("stty sane 2> {}".format(os.devnull))  # in case signalled from input_with_prefill
    if progress.progressing:
        progress(False)
    else:
        cprint()
    try:
        clear_credentials()
    except Exception:
        pass
    teardown()
    cprint(_("Submission cancelled."), "red")
    os._exit(0)
```

Reply:
We definitely can, although `submit50` isn't guaranteed to be imported since `check50` might be running locally. In which case, what would you want to print?

Reply:
Very true. Should we tweak `submit50.py` to change its output message based on `ORG`, as we do elsewhere? And then have `check50.py` call `submit50.py`'s `handler` if `config.args.local` is unset, else it uses its own and just prints "Checking cancelled." too? Slightly redundant but at least thorough for now.